doc/develop/manifest/external/arduino-core-api.rst

```diff
@@ -0,0 +1,191 @@
+
+**2. Arduino-Core-Zephyr (Zephyr-Specific Implementation)**
+
+The `Arduino-Core-Zephyr <https://github.com/zephyrproject-rtos/arduino-core-zephyr>`_ module
+provides the **Zephyr-specific implementation** of the Arduino API. This is where hardware-dependent
+Arduino functions are implemented using Zephyr's native APIs and drivers.
+
+Key characteristics:
+
+* Contains the ``cores/arduino`` directory with Zephyr implementations (``zephyrCommon.cpp``, ``zephyrSerial.cpp``, etc.)
+* Provides board-specific variants with pin mappings and Device Tree overlays
+* Includes Zephyr build system integration (CMake, Kconfig, west.yml)
+* Links to ArduinoCore-API to inherit the common implementations
+* See the `project documentation <https://github.com/zephyrproject-rtos/arduino-core-zephyr/tree/main/documentation>`_ for implementation details
+
+Features Provided
+=================
+
+Together, these components provide:
+
+* Standard Arduino API functions like ``pinMode()``, ``digitalWrite()``, ``analogRead()``, etc.
+* Support for Arduino-style ``setup()`` and ``loop()`` functions
+* Pin mapping between Arduino pin numbers and Zephyr's GPIO definitions
+* Support for common Arduino communication protocols (Serial, I2C, SPI)
+* Compatibility with existing Arduino libraries
+* Board variant support for various hardware platforms already present in Zephyr
+
+By bringing Arduino-style programming to Zephyr, this module provides a gentler learning curve
+for those transitioning from Arduino to Zephyr while still benefiting from Zephyr's advanced
+features, scalability, and broad hardware support.
+
+Usage with Zephyr
+*****************
+
+Adding the Arduino Core API to a Zephyr Project
+===============================================
+
+#. To pull in the Arduino Core for Zephyr as a Zephyr module, either add it as
+   a West project in the west.yaml file or pull it in by adding a submanifest
+   (e.g. ``zephyr/submanifests/arduino-core.yaml``) file with the following content
+   and run west update:
+
+   .. code-block:: yaml
+
+      # Arduino API repository
+      - name: Arduino-Core-Zephyr
+        path: modules/lib/arduinocore-zephyr
+        revision: main
+        url: https://github.com/zephyrproject-rtos/arduino-core-zephyr
+
+#. Run the following command to update your project:
+
+   .. code-block:: bash
+
+      west update
+
+#. For Linux users, there's an ``install.sh`` script in the module that will automatically
+   link the ArduinoCore-API. If you can't use this script, follow the manual steps below.
+
+#. Complete the core setup by linking the API folder from the ArduinoCore-API repository into
+   the arduinocore-zephyr folder:
+
+   .. code-block:: bash
+
+      git clone https://github.com/arduino/ArduinoCore-API # Any location
+      cd /<path>/<to>/<zephyrproject>/modules/lib/arduinocore-zephyr
+      ln -s /<your>/<location>/arduinocore-zephyr/api cores/arduino/.
```

Comment:
Is this supposed the be the "manual steps" mentioned in the previous bullet point? If yes it probably needs to be a sub-bullet or a `.. note` or something since otherwise it's not clear users who've run the install script can skip this
What's more, it's supposed to be instructions for non Linux users and yet it uses `ln -s` which won't be available to windows users?